### PR TITLE
september validates

### PR DESCRIPTION
**Pull request by cwulfman**
september_test_case.json, with ExternalIdentifiers

### CODE REVIEW

File: test_cases/september_test_case.json [hunk 1]
```diff
@@ -1102,7 +1106,16 @@
                     }
                 ],
                 "Label": "POTUS"
-            }
+            },
+            "ExternalIdentifier": [
```
Comment (mattbeattie) [inline test_cases/september_test_case.json]: This implementation unfortunately deviates from the one in the August EDF.

* In the August EDF, [the `ExternalIdentifier` is on the `Contest`](https://github.com/TrustTheVote-Project/NIST-1500-100-103-examples/blob/bd1fa7881a52ae530fbc86a004e0221d1c9ed0a6/test_cases/august_test_case.json#L81-L119)
* In this PR, it's being added to the `Office`

Markit is configured to look on the `Contest`, and so this update results in a failure to parse - thanks to my updated parsing logic that expects it there.

I went looking in the [profile](https://github.com/TrustTheVote-Project/NIST-1500-100-103-examples/blob/main/profile/profile.org), but couldn't find any documentation about how we agreed to do this.

Conceptually, I think it makes more sense for it to be applied to the contest. What do you think?

Comment (cwulfman) [reply test_cases/september_test_case.json]: You're right. I'll fix that.

Comment (cwulfman) [reply test_cases/september_test_case.json]: @mattbeattie , the CandidateContests now have ExternalIdentifiers; please confirm

Comment (mattbeattie) [reply test_cases/september_test_case.json]: Resolving this thread as this has been completed.

File: test_cases/september_test_case.json [hunk 2]
```diff
@@ -80,6 +89,15 @@
                 {
                     "@type": "ElectionResults.CandidateContest",
                     "@id": "recsoZy7vYhS3lbcK",
+                    "ExternalIdentifier": [
```
Comment (mattbeattie) [inline test_cases/september_test_case.json:92]: This `ExternalIdentifier` is materially different from the one in the [August EDF](https://github.com/TrustTheVote-Project/NIST-1500-100-103-examples/blob/main/test_cases/august_test_case.json#L84-L91), which looks like this:

```json
                    "ExternalIdentifier": [
                        {
                            "@type": "ElectionResults.ExternalIdentifier",
                            "Type": "other",
                            "OtherType": "https://viaf.org/viaf/",
                            "Value": "129529146"
                        }
                    ],
```

Markit's logic is built to that spec, and specifically looks for an `ExternalIdentifier` whose `Value` is `129529146`. It therefore fails to parse this EDF, as it doesn't see this as a presidential contest.

Which is correct?

Comment (cwulfman) [reply test_cases/september_test_case.json:92]: @mattbeattie this is up to you; these are not real data. I'm not sure what the "right" answer is: if the ExternaIIdentifier is a URI, then its value should be `https://viaf.org/viaf/129529146`; the strategy of splitting the context (`https://viaf.org/viaf`) from the ID (`129529146`) is commonly used -- I've used it myself -- to shorten IDs, but I think it's not a good idea in general.  But again: this is your data!

Comment (mattbeattie) [reply test_cases/september_test_case.json:92]: I agree, I like what you have here better. Let's use it. Everything's good to go on Markit!

**Review by mattbeattie — APPROVED** [review]

**Pull request merged**